src/redux/migrations.ts

```diff
@@ -1917,4 +1923,29 @@ export const migrations = {
     app: _.omit(state.app, 'numberVerified'),
   }),
   233: (state: any) => state,
+  234: (state: any) => {
+    const knownCompletedStandByTransactions: string[] = state.transactions.standbyTransactions
+      .filter((tx: StandbyTransaction) => tx.status === TransactionStatus.Complete)
+      .map((tx: TokenTransaction) => tx.transactionHash)
+
+    const flattenedTransactions = Object.values(
+      state.transactions.transactionsByNetworkId
+    ).flat() as TokenTransaction[]
```

Comment:
As `transactionsByNetworkId` were grouped by `networkId` - it should be safe to assume that Zerion can also return the same hash for two different networks, considering that it should return identical data to what we were getting with the previous data providers. I think I need to change the whole implementation of `knownCompletedTransactionsHashes` to maybe store keys like `${networkId}_${transactionHash}` or an object just like `transactionsByNetworkId` but wanted to post this here to confirm. What do you think?

Reply:
Yes I think we should store just the first page with all networks combined.
But because we need to keep the old feed, this should be a new field in the reducer.